src/role-access/containers/rejected-request-view/rejected-request-view.component.spec.ts

```diff
@@ -4,12 +4,13 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { ActivatedRoute, Router } from '@angular/router';
 import { PipesModule } from '@hmcts/ccd-case-ui-toolkit';
 import { of } from 'rxjs';
+import { RejectionReasonText } from 'src/role-access/models/enums/answer-text';
 
+import { RejectedRequestViewComponent } from '..';
 import { CaseworkerDataService, WASupportedJurisdictionsService } from '../../../work-allocation/services';
 import { getMockCaseRoles } from '../../../work-allocation/tests/utils.spec';
 import { CaseRoleDetails, RoleCategory } from '../../models';
 import { AllocateRoleService } from '../../services';
-import { RejectedRequestViewComponent } from '..';
```

Comment:
Why change this? The imports are supposed to go in order of localisation to the file in majority of code (might not be 'alphabetically' correct)